experimental/packages/exporter-trace-otlp-proto/test/traceHelper.ts

```diff
@@ -238,26 +238,24 @@ export function ensureExportTraceServiceRequestIsSet(
     'resourceSpans is missing'
   );
 
-  const resource = resourceSpans[0].resource;
+  const resource = resourceSpans?.[0].resource;
   assert.strictEqual(!!resource, true, 'resource is missing');
 
-  const instrumentationLibrarySpans =
-    resourceSpans[0].instrumentationLibrarySpans;
+  const instrumentationLibrarySpans = resourceSpans?.[0].scopeSpans;
   assert.strictEqual(
     instrumentationLibrarySpans && instrumentationLibrarySpans.length,
     1,
-    'instrumentationLibrarySpans is missing'
+    'scopeSpans is missing'
   );
 
-  const instrumentationLibrary =
-    instrumentationLibrarySpans[0].instrumentationLibrary;
+  const scope = instrumentationLibrarySpans?.[0].scope;
   assert.strictEqual(
```

Comment:
Not related to this PR, but can we write this as `assert.ok(scope, 'scope is missing`?

Reply:
Yes that would be better. :sweat_smile: 
I changed it everywhere I could find it in [87be662](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/87be6625c35dcd46f89fb8d7dee7e7f1148deac1), as I was touching these lines anyway. :slightly_smiling_face: